core/src/main/java/org/elasticsearch/rest/BytesRestResponse.java

```diff
@@ -115,11 +117,20 @@ public RestStatus status() {
         return this.status;
     }
 
+    private static final ESLogger SUPPRESSED_ERROR_LOGGER = ESLoggerFactory.getLogger("rest.suppressed");
+
     private static XContentBuilder convert(RestChannel channel, RestStatus status, Throwable t) throws IOException {
         XContentBuilder builder = channel.newErrorBuilder().startObject();
         if (t == null) {
             builder.field("error", "unknown");
         } else if (channel.detailedErrorsEnabled()) {
+            final ToXContent.Params params;
+            if (channel.request().paramAsBoolean("error_trace", false)) {
```

Comment:
shouldn't we use here `!REST_EXCEPTION_SKIP_STACK_TRACE_DEFAULT` instead of `false`